test/ui/selenium/ViewJobRoles.ts

```diff
@@ -0,0 +1,80 @@
+import { assert } from 'chai';
+import { Builder, By, WebDriver } from 'selenium-webdriver';
+import HomePage from '../Pages/HomePage';
+import LoginPage from '../Pages/LoginPage';
+import { sleep } from '../utils/utils'
+
+
+const email = process.env.LOGIN_EMAIL as string;
+const password = process.env.LOGIN_PASSWORD as string;
+
+describe('View Job Roles Tests', function () {
+    let driver: WebDriver;
+
+    before(async function () {
+        driver = await new Builder().forBrowser('chrome').build();
+    });
+
+    after(async function () {
+        if (driver) {
+            await driver.quit();
+        }
+    });
+
+    it('@roles should view job roles page', async function () {
+        this.timeout(100000);
+
+        const homepage = new HomePage(driver);
+        const loginPage = new LoginPage(driver);
+
+        await homepage.mainPage();
+        await sleep(2000);
+
+        await homepage.openLoginPage();
+        await sleep(2000);
+
+        await loginPage.enterEmail(email);
+        await loginPage.enterPassword(password);
+        await loginPage.clickLoginButton();
+        await sleep(3000);
+
+        await homepage.jobRolePage();
+        await sleep(3000);
```

Comment:
This might need a longer sleep as more jobRoles get added (over 100,000)

Reply:
I only land the job role page to verify that its there